src/modules/registrypreview/RegistryPreviewUI/MainWindow.xaml

```diff
@@ -7,31 +7,42 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:winuiex="using:WinUIEx"
     Closed="Window_Closed"
+    MinWidth="480"
+    MinHeight="320"
     mc:Ignorable="d">
     <Window.SystemBackdrop>
         <MicaBackdrop />
     </Window.SystemBackdrop>
+    
     <Grid>
         <Grid.RowDefinitions>
-            <RowDefinition Height="32" />
+            <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
         <Grid 
-            x:Name="titleBar" 
+            x:Name="titleBar"
             Grid.Row="0"
+            Height="32"
+            ColumnSpacing="16"
             IsHitTestVisible="True">
-            <StackPanel
-                Margin="8,8,8,8"
-                VerticalAlignment="Bottom"
-                Orientation="Horizontal">
-                <Image Source="app.ico"/>
-                <TextBlock
-                    x:Name="titleBarText"
-                    Margin="8,0,0,0"
-                    FontSize="12"
-                    Style="{StaticResource CaptionTextBlockStyle}"
-                    Text="{Binding ApplicationTitle}" />
-            </StackPanel>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition x:Name="LeftPaddingColumn" Width="0"/>
+                <ColumnDefinition x:Name="IconColumn" Width="Auto"/>
+                <ColumnDefinition x:Name="TitleColumn" Width="Auto"/>
+                <ColumnDefinition x:Name="RightPaddingColumn" Width="0"/>
```

Comment:
This column can be removed as it serves no purpose - only needed when having interactive elements in the titlebar area.